android-components/components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/SyncableLoginsStorage.kt

```diff
@@ -105,10 +96,8 @@ class SyncableLoginsStorage(
     val crypto by lazy { LoginsCrypto(context, securePrefs.value, this) }
 
     internal val conn by lazy {
-        // Migration must run before we initialize the database.
-        runBlocking(coroutineContext) {
-            migrateSQLCipherDBIfNeeded()
-        }
+        // We do not migrate SQLCipher anymore, we should delete it if it exists
+        deleteSQLCipherDBIfNeeded()
```

Comment:
Is there follow-up work to eventually remove this as well?

Reply:
Yeah at some point we can probably remove it, though it's less clear when that should be (anyone that upgrades from v95 on after this will have a lingering SQLCipher DB laying around unused, though it'd be a small percentage of users). This is just to ensure we have proper cleanup, this will hardly run for most people

Reply:
I have created https://bugzilla.mozilla.org/show_bug.cgi?id=1850639 to track the removal of this. Should I add this to the code? or is tracking the bug sufficient?

Reply:
Just tracking the bug should be good, thanks!